packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1726,23 +1726,60 @@ Authenticators can be broadly classified into three kinds of factors. A factor r
-| `methods`            | Array             | The Authenticator Methods which are permitted.                                                                          | `PASSWORD`, `SECURITY_QUESTION`, `SMS`, `VOICE`, `EMAIL`, `FIDO2` |
-| `hardwareProtection` | String            | Indicates whether any secrets or private keys used during authentication must be hardware protected and not exportable. | `REQUIRED`, `OPTIONAL`                                                                            |
-| `reauthenticateIn`   | String (ISO 8601) | The period after which the end-user should be reauthenticated, regardless of activity.                                  | N/A                                                                                               |
-| `inactivityPeriod`   | String (ISO 8601) | The period of inactivity after which the user should be reauthenticated.                                                | N/A                                                                                               |
+| `type`         | String            | The Verification Method type                                                         | `ASSURANCE`     |
+| `constraints`        | Array of [Constraint Object](#constraints)           | A JSON array that contains nested Authenticator Constraint objects, which are organized by the Authenticator class        | [Constraint Object](#constraints) consisting of `POSSESSION` constraint or `KNOWLEDGE` constraint or both. See [Verification Method JSON Examples](#verification-method-JSON-examples)                                                                      |
+| `reauthenticateIn`   | String (ISO 8601) | The time that the End User must re-authenticate, regardless of user activity                                  | N/A                                                                                               |
+| `inactivityPeriod`   | String (ISO 8601) | The inactivity time period after which the End User must re-authenticate                                                | N/A                                                                                               |
 
-##### Constraints
+#### Constraints
 
 Each nested constraint object is treated as a list, all of which must be satisfied. The top-level array is treated as a set, one of which must be satisfied.
 
+##### Constraints default example
+```json
+"constraints": [
+  { // object 1
+    "knowledge": {  // 1A
+      "types": [
+        "password"
+      ],
+      "reauthenticateIn": "PTOS"
+    },
+    "possession": { // 1B
+      "userPresence": "OPTIONAL"
+    }
+  },
+  { // object 2
+    "knowledge": {  // 2A 
+      "types": [
+        "password"
+      ],
+    },
+    "possession": { // 2B
+      "phishingResistant": "REQUIRED"
+    }
+  }
+]
+```
```

Comment:
Rendering of the JSON object ![Screen Shot 2021-05-27 at 10 30 44 AM](https://user-images.githubusercontent.com/48875603/119871018-de7dd780-bed6-11eb-8011-44f30c016199.png)